styles.css

```diff
@@ -0,0 +1,58 @@
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+.heading-section {
+  display: flex;
+  flex-direction: column;
+}
+
+.header {
+  width: 100%;
+  margin: 0;
+}
+
+.navigate {
+  display: flex;
+  width: 100%;
+  margin: 0;
+  justify-content: space-between;
+  color: white;
+  background-color: #3c3a39;
+}
+
+body {
+  background-color: #1c1a19;
+  margin: 0;
+}
+
+.heading1 {
+  color: #f55800;
+  font-size: 50px;
+  font-family: Crete Round, var(--font-family-common);
+}
+
+.describe {
+  font-family: Roboto, var(--font-family-common);
+  font-style: normal;
+  font-weight: 500;
+  font-size: 32px;
+  color: #fff;
+}
```

Comment:
- [ ] I would suggest we stick with the Figma template. the alignment of the page title is not well positioned.
Kindly check the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=kgjXcDYS4nMWzLhZ-0) for the `padding` and `margin` to use,  as well as the `font-size` and `font-family` that easily fit into your project for proper alignment

kindly compare what we have in the Figma template: 👇 

![body 1](https://user-images.githubusercontent.com/78634841/204930190-42592e44-f202-47c5-877c-90bd5ce1e4ef.png)

to what we have in your project: 👇 

![body](https://user-images.githubusercontent.com/78634841/204930248-9bf5eda1-9140-48c7-bd1f-90e06c1c63e7.png)